README.md

```diff
@@ -169,6 +169,7 @@ A curated list of [awesome](https://github.com/sindresorhus/awesome) Open Policy
 
 ## Kubernetes
 
+- [Meshery](https://meshery.io/) - Use Meshery to configure your multi-cluster Kubernetes environments on-premises or across clouds.
```

Comment:
Could you please change the description to say how Meshery uses OPA? That's presumably what readers of this list will want to know :) Thanks.

Reply:
Done!